### ISSUE TITLE
Tu não regula nem sua casa, quer regular uma profissão

### ISSUE THREAD

**Issue by lincolixavier** [pilhacheia/tretadev#97]
#faleiToLeve

**Comment by terremoth** [comment]
Pra quem leu e tá moscando, o Lula foi no Flow Podcast e falou de um projeto que o PT tem que é de regulamentar a profissão de desenvolvedor de software.

Fazendo isso iremos pagar mais impostos, terá de ter sindicatos, maior máquina pública pra ficar enchendo bolso de vários caras que irão ficar fazendo vários nadas durante o dia alegando que "nos protegem e protegem nossos direitos" (o que é mentira porque na maior parte das vezes fazemos acordo de prestação de serviços com contrato bem definido ou somos contratados via CLT normal) e teremos um grupo de pessoas (que não são devs) dizendo o que devs podem ou não fazer, literalmente engessando como podemos trabalhar. Claro que se tu for falar pra eles isso eles vão dizer "nããããoo, não é bem assim, pirirí pororó vamos te proteger, me paguem, somos protetores necessários" mas na prática é literalmente isso que acontece já com um monte de profissão. E pior ainda: "ditar piso salarial de dev", sendo que sabemos a quantidade de linguagens frameworks, tecnologias e formas de trabalhar que existem bem como suas ramificações, daí vão querer criar um grupinho específico do que pode e o que não pode pra cada nomenclatura, daí vários patrões vão querer pagar o piso salarial + x% em contraste com o que merecemos ganhar, tornando nossa área PIOR pra trabalhar. Não to generalizando dizendo que isso acontecerá com muitos ou todos, mas é o que a gente vê aí com essas "regulamentações de profissão" sendo que mal se vê devs que ganham bem reclamando dessa necessidade.

Que venham os hates nos comentários...

**Comment by obrennomartins** [comment]
Alguém aí já viu algum engenheiro de software com carteirinha do CREA? 

Em tese, a profissão já é regulada.

**Comment by terremoth** [comment]
> Alguém aí já viu algum engenheiro de software com carteirinha do CREA?
> 
> Em tese, a profissão já é regulada.

Sim, já vi, e quem sente a necessidade de pagar deve ter sérios problemas. Provavelmente é aquele lance do ego pra "dar carteirada" nos outros "hurrr olha aqui como sou engenheiro" 
lol

**Comment by tchaguitos** [comment]
kjkkkjk rachei pqp
